VultisigApp/VultisigApp/Views/Components/TokenSelectorDropdown.swift

```diff
@@ -105,8 +110,13 @@ struct TokenSelectorDropdown: View {
     private func getCell(for coin: Coin) -> some View {
         HStack(spacing: 12) {
             ZStack {
-                ImageView(coin.logo, size: CGSize(width: 32, height: 32))
-                    .cornerRadius(50)
+                Group {
+                    if coin.logo.hasPrefix("https://") {
```

Comment:
same

Reply:
Fixed, now we have only one component for images. All encapsulated.